routes/add_trainer/add_trainer.py

```diff
@@ -0,0 +1,38 @@
+from flask import Blueprint, render_template, url_for, redirect, flash
+from forms.forms import AddTrainerForm
+from flask_login import current_user
+from db.models import Trainer
+from globals.globals import db
+from routes.login.login import redirect_to
+
+add_trainer_blueprint = Blueprint('add_trainer', __name__)
+
+
+@add_trainer_blueprint.route('/trainer/add', methods=["GET", "POST"])
+def add_trainer():
+
+    if not current_user.is_authenticated:
+        return redirect_to('login.login')
+
+    form = AddTrainerForm()
+
+    if form.validate_on_submit():
+        trainer_name = form.name.data
+        trainer_exists_in_db = Trainer().get_trainer_by_name(trainer_name)
+
+        if trainer_exists_in_db >= 1:
```

Comment:
If we change this to exists, we can use the boolean value. If you're doing it this way, you can still say ```if trainer_exists_in_db:``` because 0 evaluates to false, everything else is true.